src/digital.cpp

```diff
@@ -0,0 +1,47 @@
+#include "digital.h"
+#include <Arduino.h>
+
+#include "STM32TimerInterrupt_Generic.h"
+// #include "const.h"
+
+// Ticker to poll input readings at fixed rate
+STM32TimerInterrupt dataReader(TIM15);
```

Comment:
TIM7 should work if you are having compile issues. Also, let's rename this to `digitalReader` or something with "digital" in its name since we also have a timer for reading analog pins.